services/execsrv/service.go

```diff
@@ -81,7 +81,9 @@ func (s *ExecService) Create(ctx context.Context, projectID string, creator stri
 
 	spec := types.SetSpecDefaultValues(params.Spec)
 
-	execID, err := s.driver.ExecCreate(ctx, projectID, image, spec, volumes, []string{params.SSHPublicKey}, nil)
+	network := types.NetworkSpec{Ports: params.Ports}
+
+	execID, err := s.driver.ExecCreate(ctx, projectID, image, spec, network, volumes, []string{params.SSHPublicKey}, nil)
```

Comment:
Further down in this file, we also need to update the `Network` field of the new exec types so it gets updated in the store